client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -162,6 +169,26 @@ export const needsCustomsForm = createSelector(
 	( state, orderId, siteId = getSelectedSiteId( state ) ) => [ getForm( state, orderId, siteId ) ]
 );
 
+export const getProductValueFromOrder = createSelector(
+	( state, productId, orderId, siteId = getSelectedSiteId( state ) ) => {
+		const order = getOrder( state, orderId, siteId );
+		if ( ! order ) {
+			return 0;
+		}
+		for ( let i = 0; i < order.line_items.length; i++ ) {
+			const item = order.line_items[ i ];
+			const id = item.variation_id || item.product_id;
+			if ( id === productId ) {
+				return round( item.total / item.quantity, 2 );
+			}
+		}
+		return 0;
```

Comment:
If none of the items in the order match the product we're looking for, why does returning `0` make sense? Is showing an error better?